meticulous/experiment.py

```diff
@@ -273,11 +273,12 @@ def _set_experiments_directory(self, experiments_directory):
         # ignore the experiment directory from git tree if not ignored yet
         try:
             with open(os.path.join(self.repo_directory, '.gitignore'), 'r') as f:
-                ignored = os.path.relpath(self.experiments_directory, self.repo_directory) in [p.strip() for p in f.readlines()]
+                ignored = os.path.relpath(self.experiments_directory, self.repo_directory) in [os.path.normpath(p.strip()) for p in f.readlines()]
```

Comment:
`os.path.normpath` takes the nice entries in the `.gitignore` and transforms all linux paths with forward slash / to windows paths with backward slash \ on windows machines. As relpath outputs paths with \ on windows, this gives a more consistent check if a file is included.